website/content/partials/gui-page-instructions/select-kv-data.mdx

```diff
@@ -1,4 +1,4 @@
-- Open the data page for your `kv` plugin:
+- Navigate to your `kv` plugin:
```

Comment:
Feel free to revert, but I removed "data" here because "metadata" and "data" are key words in kv v2 and get confused a lot. In this engine it seems helpful to remove `data` unless we're talking specifically about "secret data" or endpoints that include `/data/`.